frontend/js/components/Modal/Modal.component.js

```diff
@@ -0,0 +1,175 @@
+import Actions from '../../services/Action.service.js';
+
+class ModalDb extends HTMLElement{
+    
+//    get modalId () {
+//        return this.getAttribute('modalId');
+//    }
+
+
+//    get title () {
+//     return this.getAttribute('title');
+// }
+
+    constructor(){
+        super();
+        // this.status = true;
+        // this.openDb= Actions.getopenvalue();
+        this.modalId= Actions.getModalId();
+        
+    }
+     
+    connectedCallback(){
+        this.render();
+    }
+
+
+    render(){
+        // let {modalId,title} = this;
+        // console.log(modalId,title);
+        this.innerHTML= ` <div class="modal loadDatabaseDumpModal" id="loadDatabaseDumpModal" tabindex="-1" role="dialog"
+        aria-labelledby="exampleModalCenterTitle" aria-hidden="true">
+        <div class="modal-dialog modal-dialog-centered" role="document">
+       
+          <!-- Modal content-->
+          <div class="modal-content">
+            <div class="modal-header content-center">
+              <h5 class="modal-title modal-bg" id="exampleModalLongTitle">Load Database Dump</h5>
+              <i class="large material-icons close" data-dismiss="modal" onclick="clearModal()">cancel</i>
+            </div>
+            <div class="modal-body">
+              <!-- <form id="loadDbForm"> -->
+              <div class="form-group">
+                <label class="" for="loadDbType">Database Type</label>
+                <select class="form-control load-db-input" id="loadDbType" name="loadDbType">
+                  <option value="" style="display: none;"></option>
+                  <option class="db-option" value="mysql">MySQL</option>
```

Comment:
all strings to be mentioned separatory, do not tightly bind to the constructor

Reply:
all strings are separated and declared as CONST